miraclelinux/meta-debian-extended
# coreutils: Delete duplicate recipe

# Purpose of pull request

This recipe duplicate between meta-debian and meta-debian-extended,  
delete from meta-debian-extended.


# Test
1. Build package
2. Run image and test the package on the qemuarm64 environment

## How to test the package
1. Build qemuarm64 image
2. Run qemuarm64 image
3. Check coreutils commands

### Build qemuarm64 image
Add the following to local.conf and build qemuarm64 image.
```
IMAGE_INSTALL_append = " coreutils"
```
```
$ bitbake core-image-minimal
```

### Run qemuarm64 image
```
$ runqemu qemuarm64 nographic
```

### Check coreutils commands
```
# find / -name '*coreutils'
# ls --version
# nice --version
# printenv --version
```

## Test result
### Build package
Build succeeded.
```
$ bitbake coreutils
Loading cache: 100% |############################################################################################################################################################################################################################################| Time: 0:00:00
Loaded 2384 entries from dependency cache.
Checking Debian Release ...
Sources.xz is not changed.
Force regenerate source code information.
Parsing Debian Sources.xz ...
Parsing recipes: 100% |##########################################################################################################################################################################################################################################| Time: 0:00:00
Parsing of 1361 .bb files complete (1360 cached, 1 parsed). 2381 targets, 80 skipped, 6 masked, 0 errors.
NOTE: Resolving any missing task queue dependencies

Build Configuration:
BB_VERSION           = "1.42.0"
BUILD_SYS            = "x86_64-linux"
NATIVELSBSTRING      = "universal"
TARGET_SYS           = "aarch64-emlinux-linux"
MACHINE              = "qemuarm64"
DISTRO               = "emlinux"
DISTRO_VERSION       = "2.7"
TUNE_FEATURES        = "aarch64 armv8a crc"
TARGET_FPU           = ""
meta                 
meta-yocto-bsp       = "HEAD:d4b57c68b22027c2bedff335dee06af963e4f8a8"
meta-debian          = "HEAD:c42b7c224887847b16a0348d6d12e050827d85ea"
meta-debian-extended = "delete-coreutils-recipe:25e2e2b4ed15193c1681bbb455b2c9dcaa742d48"
meta-emlinux         = "HEAD:faf103e572ae8697df8bac3861f27555b90ed5de"
meta-emlinux-private = "HEAD:8207525289999d2c41983eb8155127b62621cdb1"

Initialising tasks: 100% |#######################################################################################################################################################################################################################################| Time: 0:00:01
Sstate summary: Wanted 10 Found 0 Missed 10 Current 417 (0% match, 97% complete)
NOTE: Executing SetScene Tasks
NOTE: Executing RunQueue Tasks
NOTE: Tasks Summary: Attempted 1970 tasks of which 1958 didn't need to be rerun and all succeeded.
```

### Package test
Commands included in coreutils are installed by meta-debian's recipe.
Checked the version information with a few commands and no problems.
```
root@qemuarm64:~# find / -name '*coreutils'
/bin/mv.coreutils
/bin/touch.coreutils
/bin/hostname.coreutils
/bin/echo.coreutils
/bin/sleep.coreutils
/bin/rmdir.coreutils
/bin/stty.coreutils
/bin/false.coreutils
/bin/stat.coreutils
/bin/mkdir.coreutils
/bin/pwd.coreutils
/bin/kill.coreutils
/bin/cp.coreutils
/bin/chgrp.coreutils
/bin/date.coreutils
/bin/chmod.coreutils
/bin/ls.coreutils
/bin/true.coreutils
/bin/mknod.coreutils
/bin/ln.coreutils
/bin/rm.coreutils
/bin/cat.coreutils
/bin/dd.coreutils
/bin/chown.coreutils
/bin/uname.coreutils
/bin/sync.coreutils
/usr/sbin/chroot.coreutils
/usr/bin/hostid.coreutils
/usr/bin/chcon.coreutils
/usr/bin/pinky.coreutils
/usr/bin/mktemp.coreutils
/usr/bin/link.coreutils
/usr/bin/printenv.coreutils
/usr/bin/env.coreutils
/usr/bin/shred.coreutils
/usr/bin/uniq.coreutils
/usr/bin/od.coreutils
/usr/bin/nohup.coreutils
/usr/bin/stdbuf.coreutils
/usr/bin/comm.coreutils
/usr/bin/split.coreutils
/usr/bin/tsort.coreutils
/usr/bin/install.coreutils
/usr/bin/sha512sum.coreutils
/usr/bin/pr.coreutils
/usr/bin/tty.coreutils
/usr/bin/tac.coreutils
/usr/bin/sha1sum.coreutils
/usr/bin/csplit.coreutils
/usr/bin/unexpand.coreutils
/usr/bin/expr.coreutils
/usr/bin/mkfifo.coreutils
/usr/bin/users.coreutils
/usr/bin/unlink.coreutils
/usr/bin/arch.coreutils
/usr/bin/df.coreutils
/usr/bin/head.coreutils
/usr/bin/realpath.coreutils
/usr/bin/timeout.coreutils
/usr/bin/ptx.coreutils
/usr/bin/cksum.coreutils
/usr/bin/uptime.coreutils
/usr/bin/tail.coreutils
/usr/bin/truncate.coreutils
/usr/bin/sha384sum.coreutils
/usr/bin/nproc.coreutils
/usr/bin/nl.coreutils
/usr/bin/test.coreutils
/usr/bin/basename.coreutils
/usr/bin/fold.coreutils
/usr/bin/fmt.coreutils
/usr/bin/readlink.coreutils
/usr/bin/expand.coreutils
/usr/bin/lbracket.coreutils
/usr/bin/dirname.coreutils
/usr/bin/join.coreutils
/usr/bin/id.coreutils
/usr/bin/sum.coreutils
/usr/bin/sha224sum.coreutils
/usr/bin/groups.coreutils
/usr/bin/md5sum.coreutils
/usr/bin/base64.coreutils
/usr/bin/dircolors.coreutils
/usr/bin/whoami.coreutils
/usr/bin/du.coreutils
/usr/bin/vdir.coreutils
/usr/bin/paste.coreutils
/usr/bin/nice.coreutils
/usr/bin/cut.coreutils
/usr/bin/runcon.coreutils
/usr/bin/sort.coreutils
/usr/bin/logname.coreutils
/usr/bin/sha256sum.coreutils
/usr/bin/shuf.coreutils
/usr/bin/seq.coreutils
/usr/bin/pathchk.coreutils
/usr/bin/dir.coreutils
/usr/bin/printf.coreutils
/usr/bin/tee.coreutils
/usr/bin/who.coreutils
/usr/bin/wc.coreutils
/usr/bin/yes.coreutils
/usr/bin/tr.coreutils
/usr/bin/factor.coreutils
/usr/lib/coreutils
root@qemuarm64:~# ls --version
ls (GNU coreutils) 8.30
Copyright (C) 2018 Free Software Foundation, Inc.
License GPLv3+: GNU GPL version 3 or later <https://gnu.org/licenses/gpl.html>.
This is free software: you are free to change and redistribute it.
There is NO WARRANTY, to the extent permitted by law.

Written by Richard M. Stallman and David MacKenzie.
root@qemuarm64:~# nice --version
nice (GNU coreutils) 8.30
Copyright (C) 2018 Free Software Foundation, Inc.
License GPLv3+: GNU GPL version 3 or later <https://gnu.org/licenses/gpl.html>.
This is free software: you are free to change and redistribute it.
There is NO WARRANTY, to the extent permitted by law.

Written by David MacKenzie.
root@qemuarm64:~# printenv --version
printenv (GNU coreutils) 8.30
Copyright (C) 2018 Free Software Foundation, Inc.
License GPLv3+: GNU GPL version 3 or later <https://gnu.org/licenses/gpl.html>.
This is free software: you are free to change and redistribute it.
There is NO WARRANTY, to the extent permitted by law.

Written by David MacKenzie and Richard Mlynarik.
```
